src/main/java/com/sprint/mission/discodeit/service/basic/BasicUserService.java

```diff
@@ -38,8 +40,9 @@ public class BasicUserService implements UserService {
   private final UserRepository userRepository;
   private final BinaryContentService binaryContentService;
   private final BinaryContentRepository binaryContentRepository;
-  private final UserStatusRepository userStatusRepository;
   private final UserMapper userMapper;
+  private final BCryptPasswordEncoder bCryptPasswordEncoder;
```

Comment:
구현체가 아닌 인터페이스에 의존하는게 좋을 것 같아요.  
```suggestion
  private final PasswordEncoder passwordEncoder;
```